fab/fabfile.py

```diff
@@ -145,6 +145,7 @@ def tag_commit(self):
             tagger={
                 'name': user.name,
                 'email': user.email,
+                'date': datetime.datetime.utcnow().strftime('%Y-%m-%dT%H:%M:%SZ'),
```

Comment:
How'd you figure out what format to use?  [This](https://github.com/sigmavirus24/github3.py/blob/500138a601f65ef3f26b32c17ab7861963933207/tests/integration/test_repos_repo.py#L422-L426) is the closest thing to documentation I was able to find.  `datetime` objects also have an `isoformat()` method that might work here.